docs/ota-client-guide/modules/ROOT/pages/change-signature-thresholds.adoc

```diff
@@ -0,0 +1,52 @@
+= Change signature thresholds
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+A threshold cryptosystem is one of the ways in which we protect your software repository. By default, the threshold number is one. This means that you can use any of the valid keys to sign your software versions. For more security, you can also change the threshold number of your Root and Targets keys, as well as delegations.
+
+*To change the thresholds of Root and Targets keys:*
+
```

Comment:
It doesn't really make sense to change the key thresholds without adding more keys. There should be a section here that explains how to add keys.

Reply:
Added.
Please review.